{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 2,
   "id": "085bb1af",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Defaulting to user installation because normal site-packages is not writeable\n",
      "Collecting mysql\n",
      "  Downloading mysql-0.0.3-py3-none-any.whl (1.2 kB)\n",
      "Collecting mysqlclient\n",
      "  Downloading mysqlclient-2.1.0.tar.gz (87 kB)\n",
      "     |████████████████████████████████| 87 kB 5.1 MB/s             \n",
      "\u001b[?25h  Preparing metadata (setup.py) ... \u001b[?25lerror\n",
      "\u001b[31m  ERROR: Command errored out with exit status 1:\n",
      "   command: /usr/bin/python3 -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '\"'\"'/tmp/pip-install-082ypm9p/mysqlclient_edb9224768904429a78219d25889beef/setup.py'\"'\"'; __file__='\"'\"'/tmp/pip-install-082ypm9p/mysqlclient_edb9224768904429a78219d25889beef/setup.py'\"'\"';f = getattr(tokenize, '\"'\"'open'\"'\"', open)(__file__) if os.path.exists(__file__) else io.StringIO('\"'\"'from setuptools import setup; setup()'\"'\"');code = f.read().replace('\"'\"'\\r\\n'\"'\"', '\"'\"'\\n'\"'\"');f.close();exec(compile(code, __file__, '\"'\"'exec'\"'\"'))' egg_info --egg-base /tmp/pip-pip-egg-info-udp57_f3\n",
      "       cwd: /tmp/pip-install-082ypm9p/mysqlclient_edb9224768904429a78219d25889beef/\n",
      "  Complete output (15 lines):\n",
      "  /bin/sh: 1: mysql_config: not found\n",
      "  /bin/sh: 1: mariadb_config: not found\n",
      "  /bin/sh: 1: mysql_config: not found\n",
      "  Traceback (most recent call last):\n",
      "    File \"<string>\", line 1, in <module>\n",
      "    File \"/tmp/pip-install-082ypm9p/mysqlclient_edb9224768904429a78219d25889beef/setup.py\", line 15, in <module>\n",
      "      metadata, options = get_config()\n",
      "    File \"/tmp/pip-install-082ypm9p/mysqlclient_edb9224768904429a78219d25889beef/setup_posix.py\", line 70, in get_config\n",
      "      libs = mysql_config(\"libs\")\n",
      "    File \"/tmp/pip-install-082ypm9p/mysqlclient_edb9224768904429a78219d25889beef/setup_posix.py\", line 31, in mysql_config\n",
      "      raise OSError(\"{} not found\".format(_mysql_config_path))\n",
      "  OSError: mysql_config not found\n",
      "  mysql_config --version\n",
      "  mariadb_config --version\n",
      "  mysql_config --libs\n",
      "  ----------------------------------------\u001b[0m\n",
      "\u001b[33mWARNING: Discarding https://files.pythonhosted.org/packages/de/79/d02be3cb942afda6c99ca207858847572e38146eb73a7c4bfe3bdf154626/mysqlclient-2.1.0.tar.gz#sha256=973235686f1b720536d417bf0a0d39b4ab3d5086b2b6ad5e6752393428c02b12 (from https://pypi.org/simple/mysqlclient/) (requires-python:>=3.5). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.\u001b[0m\n",
      "\u001b[?25h  Downloading mysqlclient-2.0.3.tar.gz (88 kB)\n",
      "     |████████████████████████████████| 88 kB 6.6 MB/s             \n",
      "\u001b[?25h  Preparing metadata (setup.py) ... \u001b[?25lerror\n",
      "\u001b[31m  ERROR: Command errored out with exit status 1:\n",
      "   command: /usr/bin/python3 -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '\"'\"'/tmp/pip-install-082ypm9p/mysqlclient_9e73c398934b499fa0b57dd0f303b21c/setup.py'\"'\"'; __file__='\"'\"'/tmp/pip-install-082ypm9p/mysqlclient_9e73c398934b499fa0b57dd0f303b21c/setup.py'\"'\"';f = getattr(tokenize, '\"'\"'open'\"'\"', open)(__file__) if os.path.exists(__file__) else io.StringIO('\"'\"'from setuptools import setup; setup()'\"'\"');code = f.read().replace('\"'\"'\\r\\n'\"'\"', '\"'\"'\\n'\"'\"');f.close();exec(compile(code, __file__, '\"'\"'exec'\"'\"'))' egg_info --egg-base /tmp/pip-pip-egg-info-uruxvw2t\n",
      "       cwd: /tmp/pip-install-082ypm9p/mysqlclient_9e73c398934b499fa0b57dd0f303b21c/\n",
      "  Complete output (15 lines):\n",
      "  /bin/sh: 1: mysql_config: not found\n",
      "  /bin/sh: 1: mariadb_config: not found\n",
      "  /bin/sh: 1: mysql_config: not found\n",
      "  Traceback (most recent call last):\n",
      "    File \"<string>\", line 1, in <module>\n",
      "    File \"/tmp/pip-install-082ypm9p/mysqlclient_9e73c398934b499fa0b57dd0f303b21c/setup.py\", line 15, in <module>\n",
      "      metadata, options = get_config()\n",
      "    File \"/tmp/pip-install-082ypm9p/mysqlclient_9e73c398934b499fa0b57dd0f303b21c/setup_posix.py\", line 70, in get_config\n",
      "      libs = mysql_config(\"libs\")\n",
      "    File \"/tmp/pip-install-082ypm9p/mysqlclient_9e73c398934b499fa0b57dd0f303b21c/setup_posix.py\", line 31, in mysql_config\n",
      "      raise OSError(\"{} not found\".format(_mysql_config_path))\n",
      "  OSError: mysql_config not found\n",
      "  mysql_config --version\n",
      "  mariadb_config --version\n",
      "  mysql_config --libs\n",
      "  ----------------------------------------\u001b[0m\n",
      "\u001b[33mWARNING: Discarding https://files.pythonhosted.org/packages/3c/df/59cd2fa5e48d0804d213bdcb1acb4d08c403b61c7ff7ed4dd4a6a2deb3f7/mysqlclient-2.0.3.tar.gz#sha256=f6ebea7c008f155baeefe16c56cd3ee6239f7a5a9ae42396c2f1860f08a7c432 (from https://pypi.org/simple/mysqlclient/) (requires-python:>=3.5). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.\u001b[0m\n",
      "\u001b[?25h  Downloading mysqlclient-2.0.2.tar.gz (88 kB)\n",
      "     |████████████████████████████████| 88 kB 12.4 MB/s            \n",
      "\u001b[?25h  Preparing metadata (setup.py) ... \u001b[?25lerror\n",
      "\u001b[31m  ERROR: Command errored out with exit status 1:\n",
      "   command: /usr/bin/python3 -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '\"'\"'/tmp/pip-install-082ypm9p/mysqlclient_b5192daf2a6c456b9e05baba7c28a944/setup.py'\"'\"'; __file__='\"'\"'/tmp/pip-install-082ypm9p/mysqlclient_b5192daf2a6c456b9e05baba7c28a944/setup.py'\"'\"';f = getattr(tokenize, '\"'\"'open'\"'\"', open)(__file__) if os.path.exists(__file__) else io.StringIO('\"'\"'from setuptools import setup; setup()'\"'\"');code = f.read().replace('\"'\"'\\r\\n'\"'\"', '\"'\"'\\n'\"'\"');f.close();exec(compile(code, __file__, '\"'\"'exec'\"'\"'))' egg_info --egg-base /tmp/pip-pip-egg-info-nbaw02lo\n",
      "       cwd: /tmp/pip-install-082ypm9p/mysqlclient_b5192daf2a6c456b9e05baba7c28a944/\n",
      "  Complete output (12 lines):\n",
      "  /bin/sh: 1: mysql_config: not found\n",
      "  /bin/sh: 1: mariadb_config: not found\n",
      "  /bin/sh: 1: mysql_config: not found\n",
      "  Traceback (most recent call last):\n",
      "    File \"<string>\", line 1, in <module>\n",
      "    File \"/tmp/pip-install-082ypm9p/mysqlclient_b5192daf2a6c456b9e05baba7c28a944/setup.py\", line 15, in <module>\n",
      "      metadata, options = get_config()\n",
      "    File \"/tmp/pip-install-082ypm9p/mysqlclient_b5192daf2a6c456b9e05baba7c28a944/setup_posix.py\", line 65, in get_config\n",
      "      libs = mysql_config(\"libs\")\n",
      "    File \"/tmp/pip-install-082ypm9p/mysqlclient_b5192daf2a6c456b9e05baba7c28a944/setup_posix.py\", line 31, in mysql_config\n",
      "      raise OSError(\"{} not found\".format(_mysql_config_path))\n",
      "  OSError: mysql_config not found\n",
      "  ----------------------------------------\u001b[0m\n",
      "\u001b[33mWARNING: Discarding https://files.pythonhosted.org/packages/0e/68/f79de0e0fcdb041783124b825771e0ba6a3d0893b7b4c4735f130e42af47/mysqlclient-2.0.2.tar.gz#sha256=8df057b08fc27d8f7106bfa997d0a21e2acef017f905f06d6fb0aa6a20d4d2b2 (from https://pypi.org/simple/mysqlclient/) (requires-python:>=3.5). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.\u001b[0m\n",
      "\u001b[?25h  Downloading mysqlclient-2.0.1.tar.gz (87 kB)\n",
      "     |████████████████████████████████| 87 kB 11.2 MB/s            \n",
      "\u001b[?25h  Preparing metadata (setup.py) ... \u001b[?25lerror\n",
      "\u001b[31m  ERROR: Command errored out with exit status 1:\n",
      "   command: /usr/bin/python3 -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '\"'\"'/tmp/pip-install-082ypm9p/mysqlclient_b90abddfc5dc4cf3a6e0d0bf7e51026c/setup.py'\"'\"'; __file__='\"'\"'/tmp/pip-install-082ypm9p/mysqlclient_b90abddfc5dc4cf3a6e0d0bf7e51026c/setup.py'\"'\"';f = getattr(tokenize, '\"'\"'open'\"'\"', open)(__file__) if os.path.exists(__file__) else io.StringIO('\"'\"'from setuptools import setup; setup()'\"'\"');code = f.read().replace('\"'\"'\\r\\n'\"'\"', '\"'\"'\\n'\"'\"');f.close();exec(compile(code, __file__, '\"'\"'exec'\"'\"'))' egg_info --egg-base /tmp/pip-pip-egg-info-sutru5bp\n",
      "       cwd: /tmp/pip-install-082ypm9p/mysqlclient_b90abddfc5dc4cf3a6e0d0bf7e51026c/\n",
      "  Complete output (12 lines):\n",
      "  /bin/sh: 1: mysql_config: not found\n",
      "  /bin/sh: 1: mariadb_config: not found\n",
      "  /bin/sh: 1: mysql_config: not found\n",
      "  Traceback (most recent call last):\n",
      "    File \"<string>\", line 1, in <module>\n",
      "    File \"/tmp/pip-install-082ypm9p/mysqlclient_b90abddfc5dc4cf3a6e0d0bf7e51026c/setup.py\", line 15, in <module>\n",
      "      metadata, options = get_config()\n",
      "    File \"/tmp/pip-install-082ypm9p/mysqlclient_b90abddfc5dc4cf3a6e0d0bf7e51026c/setup_posix.py\", line 65, in get_config\n",
      "      libs = mysql_config(\"libs\")\n",
      "    File \"/tmp/pip-install-082ypm9p/mysqlclient_b90abddfc5dc4cf3a6e0d0bf7e51026c/setup_posix.py\", line 31, in mysql_config\n",
      "      raise OSError(\"{} not found\".format(_mysql_config_path))\n",
      "  OSError: mysql_config not found\n",
      "  ----------------------------------------\u001b[0m\n",
      "\u001b[33mWARNING: Discarding https://files.pythonhosted.org/packages/a5/e1/e5f2b231c05dc51d9d87fa5066f90d1405345c54b14b0b11a1c859020f21/mysqlclient-2.0.1.tar.gz#sha256=fb2f75aea14722390d2d8ddf384ad99da708c707a96656210a7be8af20a2c5e5 (from https://pypi.org/simple/mysqlclient/) (requires-python:>=3.5). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.\u001b[0m\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[?25h  Downloading mysqlclient-2.0.0.tar.gz (87 kB)\n",
      "     |████████████████████████████████| 87 kB 9.3 MB/s             \n",
      "\u001b[?25h  Preparing metadata (setup.py) ... \u001b[?25lerror\n",
      "\u001b[31m  ERROR: Command errored out with exit status 1:\n",
      "   command: /usr/bin/python3 -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '\"'\"'/tmp/pip-install-082ypm9p/mysqlclient_2024599f35dc48228c066916a662d532/setup.py'\"'\"'; __file__='\"'\"'/tmp/pip-install-082ypm9p/mysqlclient_2024599f35dc48228c066916a662d532/setup.py'\"'\"';f = getattr(tokenize, '\"'\"'open'\"'\"', open)(__file__) if os.path.exists(__file__) else io.StringIO('\"'\"'from setuptools import setup; setup()'\"'\"');code = f.read().replace('\"'\"'\\r\\n'\"'\"', '\"'\"'\\n'\"'\"');f.close();exec(compile(code, __file__, '\"'\"'exec'\"'\"'))' egg_info --egg-base /tmp/pip-pip-egg-info-yg83tvhe\n",
      "       cwd: /tmp/pip-install-082ypm9p/mysqlclient_2024599f35dc48228c066916a662d532/\n",
      "  Complete output (12 lines):\n",
      "  /bin/sh: 1: mysql_config: not found\n",
      "  /bin/sh: 1: mariadb_config: not found\n",
      "  /bin/sh: 1: mysql_config: not found\n",
      "  Traceback (most recent call last):\n",
      "    File \"<string>\", line 1, in <module>\n",
      "    File \"/tmp/pip-install-082ypm9p/mysqlclient_2024599f35dc48228c066916a662d532/setup.py\", line 15, in <module>\n",
      "      metadata, options = get_config()\n",
      "    File \"/tmp/pip-install-082ypm9p/mysqlclient_2024599f35dc48228c066916a662d532/setup_posix.py\", line 65, in get_config\n",
      "      libs = mysql_config(\"libs\")\n",
      "    File \"/tmp/pip-install-082ypm9p/mysqlclient_2024599f35dc48228c066916a662d532/setup_posix.py\", line 31, in mysql_config\n",
      "      raise OSError(\"{} not found\".format(_mysql_config_path))\n",
      "  OSError: mysql_config not found\n",
      "  ----------------------------------------\u001b[0m\n",
      "\u001b[33mWARNING: Discarding https://files.pythonhosted.org/packages/ca/cf/dd8124bf8b3d2ae465db9c8ce35c51650c31784b095f41faf6f8cad987a0/mysqlclient-2.0.0.tar.gz#sha256=201dfcc2a3c8a0859ea399947f1af8abe74b706beba0ebac149180d9d7b0f6e8 (from https://pypi.org/simple/mysqlclient/) (requires-python:>=3.5). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.\u001b[0m\n",
      "\u001b[?25h  Downloading mysqlclient-1.4.6.tar.gz (85 kB)\n",
      "     |████████████████████████████████| 85 kB 5.9 MB/s             \n",
      "\u001b[?25h  Preparing metadata (setup.py) ... \u001b[?25lerror\n",
      "\u001b[31m  ERROR: Command errored out with exit status 1:\n",
      "   command: /usr/bin/python3 -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '\"'\"'/tmp/pip-install-082ypm9p/mysqlclient_8f728cf48b23410c9d1a18faa2507c71/setup.py'\"'\"'; __file__='\"'\"'/tmp/pip-install-082ypm9p/mysqlclient_8f728cf48b23410c9d1a18faa2507c71/setup.py'\"'\"';f = getattr(tokenize, '\"'\"'open'\"'\"', open)(__file__) if os.path.exists(__file__) else io.StringIO('\"'\"'from setuptools import setup; setup()'\"'\"');code = f.read().replace('\"'\"'\\r\\n'\"'\"', '\"'\"'\\n'\"'\"');f.close();exec(compile(code, __file__, '\"'\"'exec'\"'\"'))' egg_info --egg-base /tmp/pip-pip-egg-info-5ombcze_\n",
      "       cwd: /tmp/pip-install-082ypm9p/mysqlclient_8f728cf48b23410c9d1a18faa2507c71/\n",
      "  Complete output (12 lines):\n",
      "  /bin/sh: 1: mysql_config: not found\n",
      "  /bin/sh: 1: mariadb_config: not found\n",
      "  /bin/sh: 1: mysql_config: not found\n",
      "  Traceback (most recent call last):\n",
      "    File \"<string>\", line 1, in <module>\n",
      "    File \"/tmp/pip-install-082ypm9p/mysqlclient_8f728cf48b23410c9d1a18faa2507c71/setup.py\", line 16, in <module>\n",
      "      metadata, options = get_config()\n",
      "    File \"/tmp/pip-install-082ypm9p/mysqlclient_8f728cf48b23410c9d1a18faa2507c71/setup_posix.py\", line 61, in get_config\n",
      "      libs = mysql_config(\"libs\")\n",
      "    File \"/tmp/pip-install-082ypm9p/mysqlclient_8f728cf48b23410c9d1a18faa2507c71/setup_posix.py\", line 29, in mysql_config\n",
      "      raise EnvironmentError(\"%s not found\" % (_mysql_config_path,))\n",
      "  OSError: mysql_config not found\n",
      "  ----------------------------------------\u001b[0m\n",
      "\u001b[33mWARNING: Discarding https://files.pythonhosted.org/packages/d0/97/7326248ac8d5049968bf4ec708a5d3d4806e412a42e74160d7f266a3e03a/mysqlclient-1.4.6.tar.gz#sha256=f3fdaa9a38752a3b214a6fe79d7cae3653731a53e577821f9187e67cbecb2e16 (from https://pypi.org/simple/mysqlclient/). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.\u001b[0m\n",
      "\u001b[?25h  Downloading mysqlclient-1.4.5.tar.gz (86 kB)\n",
      "     |████████████████████████████████| 86 kB 6.3 MB/s             \n",
      "\u001b[?25h  Preparing metadata (setup.py) ... \u001b[?25lerror\n",
      "\u001b[31m  ERROR: Command errored out with exit status 1:\n",
      "   command: /usr/bin/python3 -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '\"'\"'/tmp/pip-install-082ypm9p/mysqlclient_8a46d6ab34f748f89017e830f404508a/setup.py'\"'\"'; __file__='\"'\"'/tmp/pip-install-082ypm9p/mysqlclient_8a46d6ab34f748f89017e830f404508a/setup.py'\"'\"';f = getattr(tokenize, '\"'\"'open'\"'\"', open)(__file__) if os.path.exists(__file__) else io.StringIO('\"'\"'from setuptools import setup; setup()'\"'\"');code = f.read().replace('\"'\"'\\r\\n'\"'\"', '\"'\"'\\n'\"'\"');f.close();exec(compile(code, __file__, '\"'\"'exec'\"'\"'))' egg_info --egg-base /tmp/pip-pip-egg-info-0l73_1ic\n",
      "       cwd: /tmp/pip-install-082ypm9p/mysqlclient_8a46d6ab34f748f89017e830f404508a/\n",
      "  Complete output (12 lines):\n",
      "  /bin/sh: 1: mysql_config: not found\n",
      "  /bin/sh: 1: mariadb_config: not found\n",
      "  /bin/sh: 1: mysql_config: not found\n",
      "  Traceback (most recent call last):\n",
      "    File \"<string>\", line 1, in <module>\n",
      "    File \"/tmp/pip-install-082ypm9p/mysqlclient_8a46d6ab34f748f89017e830f404508a/setup.py\", line 16, in <module>\n",
      "      metadata, options = get_config()\n",
      "    File \"/tmp/pip-install-082ypm9p/mysqlclient_8a46d6ab34f748f89017e830f404508a/setup_posix.py\", line 61, in get_config\n",
      "      libs = mysql_config(\"libs\")\n",
      "    File \"/tmp/pip-install-082ypm9p/mysqlclient_8a46d6ab34f748f89017e830f404508a/setup_posix.py\", line 29, in mysql_config\n",
      "      raise EnvironmentError(\"%s not found\" % (_mysql_config_path,))\n",
      "  OSError: mysql_config not found\n",
      "  ----------------------------------------\u001b[0m\n",
      "\u001b[33mWARNING: Discarding https://files.pythonhosted.org/packages/f8/9b/5db9a03e2088a87c26e3e4d4c7f7e8f4c2dbae610f9521cdfac15755a795/mysqlclient-1.4.5.tar.gz#sha256=e80109b0ae8d952b900b31b623181532e5e89376d707dcbeb63f99e69cefe559 (from https://pypi.org/simple/mysqlclient/). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.\u001b[0m\n",
      "\u001b[?25h  Downloading mysqlclient-1.4.4.tar.gz (86 kB)\n",
      "     |████████████████████████████████| 86 kB 8.0 MB/s             \n",
      "\u001b[?25h  Preparing metadata (setup.py) ... \u001b[?25lerror\n",
      "\u001b[31m  ERROR: Command errored out with exit status 1:\n",
      "   command: /usr/bin/python3 -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '\"'\"'/tmp/pip-install-082ypm9p/mysqlclient_90536fbc534c4d3dacd1cbf01a7657b3/setup.py'\"'\"'; __file__='\"'\"'/tmp/pip-install-082ypm9p/mysqlclient_90536fbc534c4d3dacd1cbf01a7657b3/setup.py'\"'\"';f = getattr(tokenize, '\"'\"'open'\"'\"', open)(__file__) if os.path.exists(__file__) else io.StringIO('\"'\"'from setuptools import setup; setup()'\"'\"');code = f.read().replace('\"'\"'\\r\\n'\"'\"', '\"'\"'\\n'\"'\"');f.close();exec(compile(code, __file__, '\"'\"'exec'\"'\"'))' egg_info --egg-base /tmp/pip-pip-egg-info-h57z2i0q\n",
      "       cwd: /tmp/pip-install-082ypm9p/mysqlclient_90536fbc534c4d3dacd1cbf01a7657b3/\n",
      "  Complete output (12 lines):\n",
      "  /bin/sh: 1: mysql_config: not found\n",
      "  /bin/sh: 1: mariadb_config: not found\n",
      "  /bin/sh: 1: mysql_config: not found\n",
      "  Traceback (most recent call last):\n",
      "    File \"<string>\", line 1, in <module>\n",
      "    File \"/tmp/pip-install-082ypm9p/mysqlclient_90536fbc534c4d3dacd1cbf01a7657b3/setup.py\", line 16, in <module>\n",
      "      metadata, options = get_config()\n",
      "    File \"/tmp/pip-install-082ypm9p/mysqlclient_90536fbc534c4d3dacd1cbf01a7657b3/setup_posix.py\", line 61, in get_config\n",
      "      libs = mysql_config(\"libs\")\n",
      "    File \"/tmp/pip-install-082ypm9p/mysqlclient_90536fbc534c4d3dacd1cbf01a7657b3/setup_posix.py\", line 29, in mysql_config\n",
      "      raise EnvironmentError(\"%s not found\" % (_mysql_config_path,))\n",
      "  OSError: mysql_config not found\n",
      "  ----------------------------------------\u001b[0m\n",
      "\u001b[33mWARNING: Discarding https://files.pythonhosted.org/packages/4d/38/c5f8bac9c50f3042c8f05615f84206f77f03db79781db841898fde1bb284/mysqlclient-1.4.4.tar.gz#sha256=9c737cc55a5dc8dd3583a942d5a9b21be58d16f00f5fefca4e575e7d9682e98c (from https://pypi.org/simple/mysqlclient/). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.\u001b[0m\n",
      "\u001b[?25h  Downloading mysqlclient-1.4.3.tar.gz (86 kB)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     |████████████████████████████████| 86 kB 7.7 MB/s             \n",
      "\u001b[?25h  Preparing metadata (setup.py) ... \u001b[?25lerror\n",
      "\u001b[31m  ERROR: Command errored out with exit status 1:\n",
      "   command: /usr/bin/python3 -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '\"'\"'/tmp/pip-install-082ypm9p/mysqlclient_0d700faa74df4325929f0bbc2e2f4314/setup.py'\"'\"'; __file__='\"'\"'/tmp/pip-install-082ypm9p/mysqlclient_0d700faa74df4325929f0bbc2e2f4314/setup.py'\"'\"';f = getattr(tokenize, '\"'\"'open'\"'\"', open)(__file__) if os.path.exists(__file__) else io.StringIO('\"'\"'from setuptools import setup; setup()'\"'\"');code = f.read().replace('\"'\"'\\r\\n'\"'\"', '\"'\"'\\n'\"'\"');f.close();exec(compile(code, __file__, '\"'\"'exec'\"'\"'))' egg_info --egg-base /tmp/pip-pip-egg-info-gxm730by\n",
      "       cwd: /tmp/pip-install-082ypm9p/mysqlclient_0d700faa74df4325929f0bbc2e2f4314/\n",
      "  Complete output (12 lines):\n",
      "  /bin/sh: 1: mysql_config: not found\n",
      "  /bin/sh: 1: mariadb_config: not found\n",
      "  /bin/sh: 1: mysql_config: not found\n",
      "  Traceback (most recent call last):\n",
      "    File \"<string>\", line 1, in <module>\n",
      "    File \"/tmp/pip-install-082ypm9p/mysqlclient_0d700faa74df4325929f0bbc2e2f4314/setup.py\", line 16, in <module>\n",
      "      metadata, options = get_config()\n",
      "    File \"/tmp/pip-install-082ypm9p/mysqlclient_0d700faa74df4325929f0bbc2e2f4314/setup_posix.py\", line 61, in get_config\n",
      "      libs = mysql_config(\"libs\")\n",
      "    File \"/tmp/pip-install-082ypm9p/mysqlclient_0d700faa74df4325929f0bbc2e2f4314/setup_posix.py\", line 29, in mysql_config\n",
      "      raise EnvironmentError(\"%s not found\" % (_mysql_config_path,))\n",
      "  OSError: mysql_config not found\n",
      "  ----------------------------------------\u001b[0m\n",
      "\u001b[33mWARNING: Discarding https://files.pythonhosted.org/packages/13/37/6c941aabeb78464a538b7d43266bfae645993779e3ca3e12fe82b4590f52/mysqlclient-1.4.3.tar.gz#sha256=0d3eeb97534c9f42e9b65bd35c1a35fadc2f6d7ea5d9fdbda5339a569156344b (from https://pypi.org/simple/mysqlclient/). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.\u001b[0m\n",
      "\u001b[?25h  Downloading mysqlclient-1.4.2.post1.tar.gz (85 kB)\n",
      "     |████████████████████████████████| 85 kB 7.1 MB/s             \n",
      "\u001b[?25h  Preparing metadata (setup.py) ... \u001b[?25lerror\n",
      "\u001b[31m  ERROR: Command errored out with exit status 1:\n",
      "   command: /usr/bin/python3 -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '\"'\"'/tmp/pip-install-082ypm9p/mysqlclient_6556f0c0adba4167b1c2926f7915d651/setup.py'\"'\"'; __file__='\"'\"'/tmp/pip-install-082ypm9p/mysqlclient_6556f0c0adba4167b1c2926f7915d651/setup.py'\"'\"';f = getattr(tokenize, '\"'\"'open'\"'\"', open)(__file__) if os.path.exists(__file__) else io.StringIO('\"'\"'from setuptools import setup; setup()'\"'\"');code = f.read().replace('\"'\"'\\r\\n'\"'\"', '\"'\"'\\n'\"'\"');f.close();exec(compile(code, __file__, '\"'\"'exec'\"'\"'))' egg_info --egg-base /tmp/pip-pip-egg-info-v6p6jb54\n",
      "       cwd: /tmp/pip-install-082ypm9p/mysqlclient_6556f0c0adba4167b1c2926f7915d651/\n",
      "  Complete output (10 lines):\n",
      "  /bin/sh: 1: mysql_config: not found\n",
      "  Traceback (most recent call last):\n",
      "    File \"<string>\", line 1, in <module>\n",
      "    File \"/tmp/pip-install-082ypm9p/mysqlclient_6556f0c0adba4167b1c2926f7915d651/setup.py\", line 16, in <module>\n",
      "      metadata, options = get_config()\n",
      "    File \"/tmp/pip-install-082ypm9p/mysqlclient_6556f0c0adba4167b1c2926f7915d651/setup_posix.py\", line 51, in get_config\n",
      "      libs = mysql_config(\"libs\")\n",
      "    File \"/tmp/pip-install-082ypm9p/mysqlclient_6556f0c0adba4167b1c2926f7915d651/setup_posix.py\", line 29, in mysql_config\n",
      "      raise EnvironmentError(\"%s not found\" % (_mysql_config_path,))\n",
      "  OSError: mysql_config not found\n",
      "  ----------------------------------------\u001b[0m\n",
      "\u001b[33mWARNING: Discarding https://files.pythonhosted.org/packages/f4/f1/3bb6f64ca7a429729413e6556b7ba5976df06019a5245a43d36032f1061e/mysqlclient-1.4.2.post1.tar.gz#sha256=f257d250f2675d0ef99bd318906f3cfc05cef4a2f385ea695ff32a3f04b9f9a7 (from https://pypi.org/simple/mysqlclient/). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.\u001b[0m\n",
      "\u001b[?25h  Downloading mysqlclient-1.4.2.tar.gz (85 kB)\n",
      "     |████████████████████████████████| 85 kB 7.1 MB/s             \n",
      "\u001b[?25h  Preparing metadata (setup.py) ... \u001b[?25lerror\n",
      "\u001b[31m  ERROR: Command errored out with exit status 1:\n",
      "   command: /usr/bin/python3 -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '\"'\"'/tmp/pip-install-082ypm9p/mysqlclient_8e56b3d97b6e4008a6addfe41fae7d78/setup.py'\"'\"'; __file__='\"'\"'/tmp/pip-install-082ypm9p/mysqlclient_8e56b3d97b6e4008a6addfe41fae7d78/setup.py'\"'\"';f = getattr(tokenize, '\"'\"'open'\"'\"', open)(__file__) if os.path.exists(__file__) else io.StringIO('\"'\"'from setuptools import setup; setup()'\"'\"');code = f.read().replace('\"'\"'\\r\\n'\"'\"', '\"'\"'\\n'\"'\"');f.close();exec(compile(code, __file__, '\"'\"'exec'\"'\"'))' egg_info --egg-base /tmp/pip-pip-egg-info-z20i5f2h\n",
      "       cwd: /tmp/pip-install-082ypm9p/mysqlclient_8e56b3d97b6e4008a6addfe41fae7d78/\n",
      "  Complete output (10 lines):\n",
      "  /bin/sh: 1: mysql_config: not found\n",
      "  Traceback (most recent call last):\n",
      "    File \"<string>\", line 1, in <module>\n",
      "    File \"/tmp/pip-install-082ypm9p/mysqlclient_8e56b3d97b6e4008a6addfe41fae7d78/setup.py\", line 16, in <module>\n",
      "      metadata, options = get_config()\n",
      "    File \"/tmp/pip-install-082ypm9p/mysqlclient_8e56b3d97b6e4008a6addfe41fae7d78/setup_posix.py\", line 51, in get_config\n",
      "      libs = mysql_config(\"libs\")\n",
      "    File \"/tmp/pip-install-082ypm9p/mysqlclient_8e56b3d97b6e4008a6addfe41fae7d78/setup_posix.py\", line 29, in mysql_config\n",
      "      raise EnvironmentError(\"%s not found\" % (_mysql_config_path,))\n",
      "  OSError: mysql_config not found\n",
      "  ----------------------------------------\u001b[0m\n",
      "\u001b[33mWARNING: Discarding https://files.pythonhosted.org/packages/73/79/67ddf24ac31b05d741f0ac87fa612c7a11bab1b39b8270ed7344d149f8af/mysqlclient-1.4.2.tar.gz#sha256=b95edaa41d6cc47deecabcdcbb5ab437ad9ae6d8955f5cf10d1847b37e66ef5e (from https://pypi.org/simple/mysqlclient/). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.\u001b[0m\n",
      "\u001b[?25h  Downloading mysqlclient-1.4.1.tar.gz (85 kB)\n",
      "     |████████████████████████████████| 85 kB 4.6 MB/s             \n",
      "\u001b[?25h  Preparing metadata (setup.py) ... \u001b[?25lerror\n",
      "\u001b[31m  ERROR: Command errored out with exit status 1:\n",
      "   command: /usr/bin/python3 -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '\"'\"'/tmp/pip-install-082ypm9p/mysqlclient_ff646c3c43fb452d92db9fa8d42d6bd7/setup.py'\"'\"'; __file__='\"'\"'/tmp/pip-install-082ypm9p/mysqlclient_ff646c3c43fb452d92db9fa8d42d6bd7/setup.py'\"'\"';f = getattr(tokenize, '\"'\"'open'\"'\"', open)(__file__) if os.path.exists(__file__) else io.StringIO('\"'\"'from setuptools import setup; setup()'\"'\"');code = f.read().replace('\"'\"'\\r\\n'\"'\"', '\"'\"'\\n'\"'\"');f.close();exec(compile(code, __file__, '\"'\"'exec'\"'\"'))' egg_info --egg-base /tmp/pip-pip-egg-info-qy6w6emn\n",
      "       cwd: /tmp/pip-install-082ypm9p/mysqlclient_ff646c3c43fb452d92db9fa8d42d6bd7/\n",
      "  Complete output (10 lines):\n",
      "  /bin/sh: 1: mysql_config: not found\n",
      "  Traceback (most recent call last):\n",
      "    File \"<string>\", line 1, in <module>\n",
      "    File \"/tmp/pip-install-082ypm9p/mysqlclient_ff646c3c43fb452d92db9fa8d42d6bd7/setup.py\", line 16, in <module>\n",
      "      metadata, options = get_config()\n",
      "    File \"/tmp/pip-install-082ypm9p/mysqlclient_ff646c3c43fb452d92db9fa8d42d6bd7/setup_posix.py\", line 51, in get_config\n",
      "      libs = mysql_config(\"libs\")\n",
      "    File \"/tmp/pip-install-082ypm9p/mysqlclient_ff646c3c43fb452d92db9fa8d42d6bd7/setup_posix.py\", line 29, in mysql_config\n",
      "      raise EnvironmentError(\"%s not found\" % (_mysql_config_path,))\n",
      "  OSError: mysql_config not found\n",
      "  ----------------------------------------\u001b[0m\n",
      "\u001b[33mWARNING: Discarding https://files.pythonhosted.org/packages/de/d7/919042fb7ed78a89e21c4e43d5daed90d5df18ec183bae3de493a7d9e3ed/mysqlclient-1.4.1.tar.gz#sha256=a62220410e26ce2d2ff94dd0138c3ecfb91db634464a9afb4c8e6b50f0a67e00 (from https://pypi.org/simple/mysqlclient/). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.\u001b[0m\n",
      "\u001b[?25h  Downloading mysqlclient-1.4.0.tar.gz (85 kB)\n",
      "     |████████████████████████████████| 85 kB 6.5 MB/s             \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[?25h  Preparing metadata (setup.py) ... \u001b[?25lerror\n",
      "\u001b[31m  ERROR: Command errored out with exit status 1:\n",
      "   command: /usr/bin/python3 -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '\"'\"'/tmp/pip-install-082ypm9p/mysqlclient_4681bf79d4e24163a58c4ad89aa7894a/setup.py'\"'\"'; __file__='\"'\"'/tmp/pip-install-082ypm9p/mysqlclient_4681bf79d4e24163a58c4ad89aa7894a/setup.py'\"'\"';f = getattr(tokenize, '\"'\"'open'\"'\"', open)(__file__) if os.path.exists(__file__) else io.StringIO('\"'\"'from setuptools import setup; setup()'\"'\"');code = f.read().replace('\"'\"'\\r\\n'\"'\"', '\"'\"'\\n'\"'\"');f.close();exec(compile(code, __file__, '\"'\"'exec'\"'\"'))' egg_info --egg-base /tmp/pip-pip-egg-info-vq387y9l\n",
      "       cwd: /tmp/pip-install-082ypm9p/mysqlclient_4681bf79d4e24163a58c4ad89aa7894a/\n",
      "  Complete output (10 lines):\n",
      "  /bin/sh: 1: mysql_config: not found\n",
      "  Traceback (most recent call last):\n",
      "    File \"<string>\", line 1, in <module>\n",
      "    File \"/tmp/pip-install-082ypm9p/mysqlclient_4681bf79d4e24163a58c4ad89aa7894a/setup.py\", line 16, in <module>\n",
      "      metadata, options = get_config()\n",
      "    File \"/tmp/pip-install-082ypm9p/mysqlclient_4681bf79d4e24163a58c4ad89aa7894a/setup_posix.py\", line 51, in get_config\n",
      "      libs = mysql_config(\"libs\")\n",
      "    File \"/tmp/pip-install-082ypm9p/mysqlclient_4681bf79d4e24163a58c4ad89aa7894a/setup_posix.py\", line 29, in mysql_config\n",
      "      raise EnvironmentError(\"%s not found\" % (_mysql_config_path,))\n",
      "  OSError: mysql_config not found\n",
      "  ----------------------------------------\u001b[0m\n",
      "\u001b[33mWARNING: Discarding https://files.pythonhosted.org/packages/05/73/fac741eef89b0257fb1807de80e96474f40d52739cdbc32c90c4a793e79d/mysqlclient-1.4.0.tar.gz#sha256=efeaacd04feea7d44fef2a5adf4df335b438458f8b89144307dc5a90025761a5 (from https://pypi.org/simple/mysqlclient/). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.\u001b[0m\n",
      "\u001b[?25h  Downloading mysqlclient-1.3.14.tar.gz (91 kB)\n",
      "     |████████████████████████████████| 91 kB 5.5 MB/s             \n",
      "\u001b[?25h  Preparing metadata (setup.py) ... \u001b[?25lerror\n",
      "\u001b[31m  ERROR: Command errored out with exit status 1:\n",
      "   command: /usr/bin/python3 -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '\"'\"'/tmp/pip-install-082ypm9p/mysqlclient_59e12f0a6f1948babd68110e19ebf7bc/setup.py'\"'\"'; __file__='\"'\"'/tmp/pip-install-082ypm9p/mysqlclient_59e12f0a6f1948babd68110e19ebf7bc/setup.py'\"'\"';f = getattr(tokenize, '\"'\"'open'\"'\"', open)(__file__) if os.path.exists(__file__) else io.StringIO('\"'\"'from setuptools import setup; setup()'\"'\"');code = f.read().replace('\"'\"'\\r\\n'\"'\"', '\"'\"'\\n'\"'\"');f.close();exec(compile(code, __file__, '\"'\"'exec'\"'\"'))' egg_info --egg-base /tmp/pip-pip-egg-info-2o513h1w\n",
      "       cwd: /tmp/pip-install-082ypm9p/mysqlclient_59e12f0a6f1948babd68110e19ebf7bc/\n",
      "  Complete output (10 lines):\n",
      "  /bin/sh: 1: mysql_config: not found\n",
      "  Traceback (most recent call last):\n",
      "    File \"<string>\", line 1, in <module>\n",
      "    File \"/tmp/pip-install-082ypm9p/mysqlclient_59e12f0a6f1948babd68110e19ebf7bc/setup.py\", line 16, in <module>\n",
      "      metadata, options = get_config()\n",
      "    File \"/tmp/pip-install-082ypm9p/mysqlclient_59e12f0a6f1948babd68110e19ebf7bc/setup_posix.py\", line 53, in get_config\n",
      "      libs = mysql_config(\"libs_r\")\n",
      "    File \"/tmp/pip-install-082ypm9p/mysqlclient_59e12f0a6f1948babd68110e19ebf7bc/setup_posix.py\", line 28, in mysql_config\n",
      "      raise EnvironmentError(\"%s not found\" % (mysql_config.path,))\n",
      "  OSError: mysql_config not found\n",
      "  ----------------------------------------\u001b[0m\n",
      "\u001b[33mWARNING: Discarding https://files.pythonhosted.org/packages/f7/a2/1230ebbb4b91f42ad6b646e59eb8855559817ad5505d81c1ca2b5a216040/mysqlclient-1.3.14.tar.gz#sha256=3981ae9ce545901a36a8b7aed76ed02960a429f75dc53b7ad77fb2f9ab7cd56b (from https://pypi.org/simple/mysqlclient/). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.\u001b[0m\n",
      "\u001b[?25h  Downloading mysqlclient-1.3.13.tar.gz (90 kB)\n",
      "     |████████████████████████████████| 90 kB 8.9 MB/s             \n",
      "\u001b[?25h  Preparing metadata (setup.py) ... \u001b[?25lerror\n",
      "\u001b[31m  ERROR: Command errored out with exit status 1:\n",
      "   command: /usr/bin/python3 -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '\"'\"'/tmp/pip-install-082ypm9p/mysqlclient_ffba4482e2744c989309bcd07a2acf3f/setup.py'\"'\"'; __file__='\"'\"'/tmp/pip-install-082ypm9p/mysqlclient_ffba4482e2744c989309bcd07a2acf3f/setup.py'\"'\"';f = getattr(tokenize, '\"'\"'open'\"'\"', open)(__file__) if os.path.exists(__file__) else io.StringIO('\"'\"'from setuptools import setup; setup()'\"'\"');code = f.read().replace('\"'\"'\\r\\n'\"'\"', '\"'\"'\\n'\"'\"');f.close();exec(compile(code, __file__, '\"'\"'exec'\"'\"'))' egg_info --egg-base /tmp/pip-pip-egg-info-61mko8pe\n",
      "       cwd: /tmp/pip-install-082ypm9p/mysqlclient_ffba4482e2744c989309bcd07a2acf3f/\n",
      "  Complete output (10 lines):\n",
      "  /bin/sh: 1: mysql_config: not found\n",
      "  Traceback (most recent call last):\n",
      "    File \"<string>\", line 1, in <module>\n",
      "    File \"/tmp/pip-install-082ypm9p/mysqlclient_ffba4482e2744c989309bcd07a2acf3f/setup.py\", line 18, in <module>\n",
      "      metadata, options = get_config()\n",
      "    File \"/tmp/pip-install-082ypm9p/mysqlclient_ffba4482e2744c989309bcd07a2acf3f/setup_posix.py\", line 53, in get_config\n",
      "      libs = mysql_config(\"libs_r\")\n",
      "    File \"/tmp/pip-install-082ypm9p/mysqlclient_ffba4482e2744c989309bcd07a2acf3f/setup_posix.py\", line 28, in mysql_config\n",
      "      raise EnvironmentError(\"%s not found\" % (mysql_config.path,))\n",
      "  OSError: mysql_config not found\n",
      "  ----------------------------------------\u001b[0m\n",
      "\u001b[33mWARNING: Discarding https://files.pythonhosted.org/packages/ec/fd/83329b9d3e14f7344d1cb31f128e6dbba70c5975c9e57896815dbb1988ad/mysqlclient-1.3.13.tar.gz#sha256=ff8ee1be84215e6c30a746b728c41eb0701a46ca76e343af445b35ce6250644f (from https://pypi.org/simple/mysqlclient/). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.\u001b[0m\n",
      "\u001b[?25h  Downloading mysqlclient-1.3.12.tar.gz (89 kB)\n",
      "     |████████████████████████████████| 89 kB 15.4 MB/s            \n",
      "\u001b[?25h  Preparing metadata (setup.py) ... \u001b[?25lerror\n",
      "\u001b[31m  ERROR: Command errored out with exit status 1:\n",
      "   command: /usr/bin/python3 -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '\"'\"'/tmp/pip-install-082ypm9p/mysqlclient_09aeb98971dd4d8f97654c536137f8c1/setup.py'\"'\"'; __file__='\"'\"'/tmp/pip-install-082ypm9p/mysqlclient_09aeb98971dd4d8f97654c536137f8c1/setup.py'\"'\"';f = getattr(tokenize, '\"'\"'open'\"'\"', open)(__file__) if os.path.exists(__file__) else io.StringIO('\"'\"'from setuptools import setup; setup()'\"'\"');code = f.read().replace('\"'\"'\\r\\n'\"'\"', '\"'\"'\\n'\"'\"');f.close();exec(compile(code, __file__, '\"'\"'exec'\"'\"'))' egg_info --egg-base /tmp/pip-pip-egg-info-0m70kndk\n",
      "       cwd: /tmp/pip-install-082ypm9p/mysqlclient_09aeb98971dd4d8f97654c536137f8c1/\n",
      "  Complete output (10 lines):\n",
      "  /bin/sh: 1: mysql_config: not found\n",
      "  Traceback (most recent call last):\n",
      "    File \"<string>\", line 1, in <module>\n",
      "    File \"/tmp/pip-install-082ypm9p/mysqlclient_09aeb98971dd4d8f97654c536137f8c1/setup.py\", line 17, in <module>\n",
      "      metadata, options = get_config()\n",
      "    File \"/tmp/pip-install-082ypm9p/mysqlclient_09aeb98971dd4d8f97654c536137f8c1/setup_posix.py\", line 44, in get_config\n",
      "      libs = mysql_config(\"libs_r\")\n",
      "    File \"/tmp/pip-install-082ypm9p/mysqlclient_09aeb98971dd4d8f97654c536137f8c1/setup_posix.py\", line 26, in mysql_config\n",
      "      raise EnvironmentError(\"%s not found\" % (mysql_config.path,))\n",
      "  OSError: mysql_config not found\n",
      "  ----------------------------------------\u001b[0m\n",
      "\u001b[33mWARNING: Discarding https://files.pythonhosted.org/packages/6f/86/bad31f1c1bb0cc99e88ca2adb7cb5c71f7a6540c1bb001480513de76a931/mysqlclient-1.3.12.tar.gz#sha256=2d9ec33de39f4d9c64ad7322ede0521d85829ce36a76f9dd3d6ab76a9c8648e5 (from https://pypi.org/simple/mysqlclient/). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.\u001b[0m\n",
      "\u001b[?25h  Downloading mysqlclient-1.3.11.tar.gz (89 kB)\n",
      "     |████████████████████████████████| 89 kB 14.1 MB/s            \n",
      "\u001b[?25h  Preparing metadata (setup.py) ... \u001b[?25lerror\n",
      "\u001b[31m  ERROR: Command errored out with exit status 1:\n",
      "   command: /usr/bin/python3 -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '\"'\"'/tmp/pip-install-082ypm9p/mysqlclient_85a074b454b0415f8c9373ad42d466ef/setup.py'\"'\"'; __file__='\"'\"'/tmp/pip-install-082ypm9p/mysqlclient_85a074b454b0415f8c9373ad42d466ef/setup.py'\"'\"';f = getattr(tokenize, '\"'\"'open'\"'\"', open)(__file__) if os.path.exists(__file__) else io.StringIO('\"'\"'from setuptools import setup; setup()'\"'\"');code = f.read().replace('\"'\"'\\r\\n'\"'\"', '\"'\"'\\n'\"'\"');f.close();exec(compile(code, __file__, '\"'\"'exec'\"'\"'))' egg_info --egg-base /tmp/pip-pip-egg-info-eu8sao7x\n",
      "       cwd: /tmp/pip-install-082ypm9p/mysqlclient_85a074b454b0415f8c9373ad42d466ef/\n",
      "  Complete output (10 lines):\n",
      "  /bin/sh: 1: mysql_config: not found\n",
      "  Traceback (most recent call last):\n",
      "    File \"<string>\", line 1, in <module>\n",
      "    File \"/tmp/pip-install-082ypm9p/mysqlclient_85a074b454b0415f8c9373ad42d466ef/setup.py\", line 17, in <module>\n",
      "      metadata, options = get_config()\n",
      "    File \"/tmp/pip-install-082ypm9p/mysqlclient_85a074b454b0415f8c9373ad42d466ef/setup_posix.py\", line 44, in get_config\n",
      "      libs = mysql_config(\"libs_r\")\n",
      "    File \"/tmp/pip-install-082ypm9p/mysqlclient_85a074b454b0415f8c9373ad42d466ef/setup_posix.py\", line 26, in mysql_config\n",
      "      raise EnvironmentError(\"%s not found\" % (mysql_config.path,))\n",
      "  OSError: mysql_config not found\n",
      "  ----------------------------------------\u001b[0m\n",
      "\u001b[33mWARNING: Discarding https://files.pythonhosted.org/packages/6a/4d/856d1848e9b7fa8577a31c0188592f472e5cbc94f588c2a40ff930533d73/mysqlclient-1.3.11.tar.gz#sha256=9a79b38843d61f651266ff16b9fdcc8830d21f0fb852e4406a19e9a715bdf3e2 (from https://pypi.org/simple/mysqlclient/). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.\u001b[0m\n",
      "\u001b[?25h  Downloading mysqlclient-1.3.10.tar.gz (82 kB)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     |████████████████████████████████| 82 kB 379 kB/s             \n",
      "\u001b[?25h  Preparing metadata (setup.py) ... \u001b[?25lerror\n",
      "\u001b[31m  ERROR: Command errored out with exit status 1:\n",
      "   command: /usr/bin/python3 -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '\"'\"'/tmp/pip-install-082ypm9p/mysqlclient_031aed8e928b4a34a629ae4af51517dd/setup.py'\"'\"'; __file__='\"'\"'/tmp/pip-install-082ypm9p/mysqlclient_031aed8e928b4a34a629ae4af51517dd/setup.py'\"'\"';f = getattr(tokenize, '\"'\"'open'\"'\"', open)(__file__) if os.path.exists(__file__) else io.StringIO('\"'\"'from setuptools import setup; setup()'\"'\"');code = f.read().replace('\"'\"'\\r\\n'\"'\"', '\"'\"'\\n'\"'\"');f.close();exec(compile(code, __file__, '\"'\"'exec'\"'\"'))' egg_info --egg-base /tmp/pip-pip-egg-info-6t977jfz\n",
      "       cwd: /tmp/pip-install-082ypm9p/mysqlclient_031aed8e928b4a34a629ae4af51517dd/\n",
      "  Complete output (10 lines):\n",
      "  /bin/sh: 1: mysql_config: not found\n",
      "  Traceback (most recent call last):\n",
      "    File \"<string>\", line 1, in <module>\n",
      "    File \"/tmp/pip-install-082ypm9p/mysqlclient_031aed8e928b4a34a629ae4af51517dd/setup.py\", line 17, in <module>\n",
      "      metadata, options = get_config()\n",
      "    File \"/tmp/pip-install-082ypm9p/mysqlclient_031aed8e928b4a34a629ae4af51517dd/setup_posix.py\", line 44, in get_config\n",
      "      libs = mysql_config(\"libs_r\")\n",
      "    File \"/tmp/pip-install-082ypm9p/mysqlclient_031aed8e928b4a34a629ae4af51517dd/setup_posix.py\", line 26, in mysql_config\n",
      "      raise EnvironmentError(\"%s not found\" % (mysql_config.path,))\n",
      "  OSError: mysql_config not found\n",
      "  ----------------------------------------\u001b[0m\n",
      "\u001b[33mWARNING: Discarding https://files.pythonhosted.org/packages/40/9b/0bc869f290b8f49a99b8d97927f57126a5d1befcf8bac92c60dc855f2523/mysqlclient-1.3.10.tar.gz#sha256=452aaf9d2ba1a4fb85c134c88f9b1857852deed63b3b13255d7a65d2c1297262 (from https://pypi.org/simple/mysqlclient/). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.\u001b[0m\n",
      "\u001b[?25h  Downloading mysqlclient-1.3.9.tar.gz (81 kB)\n",
      "     |████████████████████████████████| 81 kB 13.7 MB/s            \n",
      "\u001b[?25h  Preparing metadata (setup.py) ... \u001b[?25lerror\n",
      "\u001b[31m  ERROR: Command errored out with exit status 1:\n",
      "   command: /usr/bin/python3 -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '\"'\"'/tmp/pip-install-082ypm9p/mysqlclient_525f33f544674d839a97482d422c9a57/setup.py'\"'\"'; __file__='\"'\"'/tmp/pip-install-082ypm9p/mysqlclient_525f33f544674d839a97482d422c9a57/setup.py'\"'\"';f = getattr(tokenize, '\"'\"'open'\"'\"', open)(__file__) if os.path.exists(__file__) else io.StringIO('\"'\"'from setuptools import setup; setup()'\"'\"');code = f.read().replace('\"'\"'\\r\\n'\"'\"', '\"'\"'\\n'\"'\"');f.close();exec(compile(code, __file__, '\"'\"'exec'\"'\"'))' egg_info --egg-base /tmp/pip-pip-egg-info-hxsm6ag7\n",
      "       cwd: /tmp/pip-install-082ypm9p/mysqlclient_525f33f544674d839a97482d422c9a57/\n",
      "  Complete output (10 lines):\n",
      "  /bin/sh: 1: mysql_config: not found\n",
      "  Traceback (most recent call last):\n",
      "    File \"<string>\", line 1, in <module>\n",
      "    File \"/tmp/pip-install-082ypm9p/mysqlclient_525f33f544674d839a97482d422c9a57/setup.py\", line 17, in <module>\n",
      "      metadata, options = get_config()\n",
      "    File \"/tmp/pip-install-082ypm9p/mysqlclient_525f33f544674d839a97482d422c9a57/setup_posix.py\", line 44, in get_config\n",
      "      libs = mysql_config(\"libs_r\")\n",
      "    File \"/tmp/pip-install-082ypm9p/mysqlclient_525f33f544674d839a97482d422c9a57/setup_posix.py\", line 26, in mysql_config\n",
      "      raise EnvironmentError(\"%s not found\" % (mysql_config.path,))\n",
      "  OSError: mysql_config not found\n",
      "  ----------------------------------------\u001b[0m\n",
      "\u001b[33mWARNING: Discarding https://files.pythonhosted.org/packages/db/f5/c8e1657985c31dda16e434edf5257c31572fa5faacd7e48b1618390e4b18/mysqlclient-1.3.9.tar.gz#sha256=990ccf1e1f15b9a291b811c993dc1c216566714bc14e6581b38665bd61c28c99 (from https://pypi.org/simple/mysqlclient/). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.\u001b[0m\n",
      "\u001b[?25h  Downloading mysqlclient-1.3.8.tar.gz (81 kB)\n",
      "     |████████████████████████████████| 81 kB 15.3 MB/s            \n",
      "\u001b[?25h  Preparing metadata (setup.py) ... \u001b[?25lerror\n",
      "\u001b[31m  ERROR: Command errored out with exit status 1:\n",
      "   command: /usr/bin/python3 -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '\"'\"'/tmp/pip-install-082ypm9p/mysqlclient_967b567ddfbf482db49ad5e31135c06d/setup.py'\"'\"'; __file__='\"'\"'/tmp/pip-install-082ypm9p/mysqlclient_967b567ddfbf482db49ad5e31135c06d/setup.py'\"'\"';f = getattr(tokenize, '\"'\"'open'\"'\"', open)(__file__) if os.path.exists(__file__) else io.StringIO('\"'\"'from setuptools import setup; setup()'\"'\"');code = f.read().replace('\"'\"'\\r\\n'\"'\"', '\"'\"'\\n'\"'\"');f.close();exec(compile(code, __file__, '\"'\"'exec'\"'\"'))' egg_info --egg-base /tmp/pip-pip-egg-info-bj3pietj\n",
      "       cwd: /tmp/pip-install-082ypm9p/mysqlclient_967b567ddfbf482db49ad5e31135c06d/\n",
      "  Complete output (10 lines):\n",
      "  /bin/sh: 1: mysql_config: not found\n",
      "  Traceback (most recent call last):\n",
      "    File \"<string>\", line 1, in <module>\n",
      "    File \"/tmp/pip-install-082ypm9p/mysqlclient_967b567ddfbf482db49ad5e31135c06d/setup.py\", line 17, in <module>\n",
      "      metadata, options = get_config()\n",
      "    File \"/tmp/pip-install-082ypm9p/mysqlclient_967b567ddfbf482db49ad5e31135c06d/setup_posix.py\", line 44, in get_config\n",
      "      libs = mysql_config(\"libs_r\")\n",
      "    File \"/tmp/pip-install-082ypm9p/mysqlclient_967b567ddfbf482db49ad5e31135c06d/setup_posix.py\", line 26, in mysql_config\n",
      "      raise EnvironmentError(\"%s not found\" % (mysql_config.path,))\n",
      "  OSError: mysql_config not found\n",
      "  ----------------------------------------\u001b[0m\n",
      "\u001b[33mWARNING: Discarding https://files.pythonhosted.org/packages/31/ce/7ea049b3d5929b3cce3104967ccca218a9f054517f0d15dfdaf4e76da2a0/mysqlclient-1.3.8.tar.gz#sha256=c64bc386c0b771ed6ff76de0f52e0f957f98d9e13c83b0ddb821afdbe3f5d970 (from https://pypi.org/simple/mysqlclient/). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.\u001b[0m\n",
      "\u001b[?25h  Downloading mysqlclient-1.3.7.tar.gz (79 kB)\n",
      "     |████████████████████████████████| 79 kB 9.4 MB/s             \n",
      "\u001b[?25h  Preparing metadata (setup.py) ... \u001b[?25lerror\n",
      "\u001b[31m  ERROR: Command errored out with exit status 1:\n",
      "   command: /usr/bin/python3 -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '\"'\"'/tmp/pip-install-082ypm9p/mysqlclient_950c525ff20b452fb129f2d7c25cfd3a/setup.py'\"'\"'; __file__='\"'\"'/tmp/pip-install-082ypm9p/mysqlclient_950c525ff20b452fb129f2d7c25cfd3a/setup.py'\"'\"';f = getattr(tokenize, '\"'\"'open'\"'\"', open)(__file__) if os.path.exists(__file__) else io.StringIO('\"'\"'from setuptools import setup; setup()'\"'\"');code = f.read().replace('\"'\"'\\r\\n'\"'\"', '\"'\"'\\n'\"'\"');f.close();exec(compile(code, __file__, '\"'\"'exec'\"'\"'))' egg_info --egg-base /tmp/pip-pip-egg-info-e7fvm56r\n",
      "       cwd: /tmp/pip-install-082ypm9p/mysqlclient_950c525ff20b452fb129f2d7c25cfd3a/\n",
      "  Complete output (10 lines):\n",
      "  /bin/sh: 1: mysql_config: not found\n",
      "  Traceback (most recent call last):\n",
      "    File \"<string>\", line 1, in <module>\n",
      "    File \"/tmp/pip-install-082ypm9p/mysqlclient_950c525ff20b452fb129f2d7c25cfd3a/setup.py\", line 17, in <module>\n",
      "      metadata, options = get_config()\n",
      "    File \"/tmp/pip-install-082ypm9p/mysqlclient_950c525ff20b452fb129f2d7c25cfd3a/setup_posix.py\", line 44, in get_config\n",
      "      libs = mysql_config(\"libs_r\")\n",
      "    File \"/tmp/pip-install-082ypm9p/mysqlclient_950c525ff20b452fb129f2d7c25cfd3a/setup_posix.py\", line 26, in mysql_config\n",
      "      raise EnvironmentError(\"%s not found\" % (mysql_config.path,))\n",
      "  OSError: mysql_config not found\n",
      "  ----------------------------------------\u001b[0m\n",
      "\u001b[33mWARNING: Discarding https://files.pythonhosted.org/packages/74/ff/4e964e20b559e55d7afa60fbccc6a560f2adf289813bd3d7eb4eb8a87093/mysqlclient-1.3.7.tar.gz#sha256=c74a83b4cb2933d0e43370117eeebdfa03077ae72686d2df43d31879267f1f1b (from https://pypi.org/simple/mysqlclient/). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.\u001b[0m\n",
      "\u001b[?25h  Downloading mysqlclient-1.3.6.tar.gz (78 kB)\n",
      "     |████████████████████████████████| 78 kB 10.8 MB/s            \n",
      "\u001b[?25h  Preparing metadata (setup.py) ... \u001b[?25lerror\n",
      "\u001b[31m  ERROR: Command errored out with exit status 1:\n",
      "   command: /usr/bin/python3 -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '\"'\"'/tmp/pip-install-082ypm9p/mysqlclient_c56743a864af4cf7b3d1795e0f912172/setup.py'\"'\"'; __file__='\"'\"'/tmp/pip-install-082ypm9p/mysqlclient_c56743a864af4cf7b3d1795e0f912172/setup.py'\"'\"';f = getattr(tokenize, '\"'\"'open'\"'\"', open)(__file__) if os.path.exists(__file__) else io.StringIO('\"'\"'from setuptools import setup; setup()'\"'\"');code = f.read().replace('\"'\"'\\r\\n'\"'\"', '\"'\"'\\n'\"'\"');f.close();exec(compile(code, __file__, '\"'\"'exec'\"'\"'))' egg_info --egg-base /tmp/pip-pip-egg-info-qvlpwgdc\n",
      "       cwd: /tmp/pip-install-082ypm9p/mysqlclient_c56743a864af4cf7b3d1795e0f912172/\n",
      "  Complete output (10 lines):\n",
      "  /bin/sh: 1: mysql_config: not found\n",
      "  Traceback (most recent call last):\n",
      "    File \"<string>\", line 1, in <module>\n",
      "    File \"/tmp/pip-install-082ypm9p/mysqlclient_c56743a864af4cf7b3d1795e0f912172/setup.py\", line 17, in <module>\n",
      "      metadata, options = get_config()\n",
      "    File \"/tmp/pip-install-082ypm9p/mysqlclient_c56743a864af4cf7b3d1795e0f912172/setup_posix.py\", line 47, in get_config\n",
      "      libs = mysql_config(\"libs_r\")\n",
      "    File \"/tmp/pip-install-082ypm9p/mysqlclient_c56743a864af4cf7b3d1795e0f912172/setup_posix.py\", line 29, in mysql_config\n",
      "      raise EnvironmentError(\"%s not found\" % (mysql_config.path,))\n",
      "  OSError: mysql_config not found\n",
      "  ----------------------------------------\u001b[0m\n",
      "\u001b[33mWARNING: Discarding https://files.pythonhosted.org/packages/09/29/6648563af4b45798ba667f17ea55166d7b0ed8717937c06977a1fdbe7df6/mysqlclient-1.3.6.tar.gz#sha256=1f63a8ab111530a0ff9acaeb118074f11e7564b7c124f2e7fc56fcac641aecf2 (from https://pypi.org/simple/mysqlclient/). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.\u001b[0m\n",
      "\u001b[?25h  Downloading mysqlclient-1.3.5.tar.gz (77 kB)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "     |████████████████████████████████| 77 kB 10.7 MB/s            \n",
      "\u001b[?25h  Preparing metadata (setup.py) ... \u001b[?25lerror\n",
      "\u001b[31m  ERROR: Command errored out with exit status 1:\n",
      "   command: /usr/bin/python3 -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '\"'\"'/tmp/pip-install-082ypm9p/mysqlclient_5ea86a26c52b463a8feba09bb37d16a4/setup.py'\"'\"'; __file__='\"'\"'/tmp/pip-install-082ypm9p/mysqlclient_5ea86a26c52b463a8feba09bb37d16a4/setup.py'\"'\"';f = getattr(tokenize, '\"'\"'open'\"'\"', open)(__file__) if os.path.exists(__file__) else io.StringIO('\"'\"'from setuptools import setup; setup()'\"'\"');code = f.read().replace('\"'\"'\\r\\n'\"'\"', '\"'\"'\\n'\"'\"');f.close();exec(compile(code, __file__, '\"'\"'exec'\"'\"'))' egg_info --egg-base /tmp/pip-pip-egg-info-vntxmwxr\n",
      "       cwd: /tmp/pip-install-082ypm9p/mysqlclient_5ea86a26c52b463a8feba09bb37d16a4/\n",
      "  Complete output (10 lines):\n",
      "  /bin/sh: 1: mysql_config: not found\n",
      "  Traceback (most recent call last):\n",
      "    File \"<string>\", line 1, in <module>\n",
      "    File \"/tmp/pip-install-082ypm9p/mysqlclient_5ea86a26c52b463a8feba09bb37d16a4/setup.py\", line 17, in <module>\n",
      "      metadata, options = get_config()\n",
      "    File \"/tmp/pip-install-082ypm9p/mysqlclient_5ea86a26c52b463a8feba09bb37d16a4/setup_posix.py\", line 47, in get_config\n",
      "      libs = mysql_config(\"libs_r\")\n",
      "    File \"/tmp/pip-install-082ypm9p/mysqlclient_5ea86a26c52b463a8feba09bb37d16a4/setup_posix.py\", line 29, in mysql_config\n",
      "      raise EnvironmentError(\"%s not found\" % (mysql_config.path,))\n",
      "  OSError: mysql_config not found\n",
      "  ----------------------------------------\u001b[0m\n",
      "\u001b[33mWARNING: Discarding https://files.pythonhosted.org/packages/82/ea/336daf6c7baa564d0c434ab748218d1a7dc61dfc7066e98b72afd24fd2b9/mysqlclient-1.3.5.tar.gz#sha256=80ea68f6f3c2cfbf35c91adde2b40de1f2c31132118d2dcee048c72423f678bf (from https://pypi.org/simple/mysqlclient/). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.\u001b[0m\n",
      "\u001b[?25h  Downloading mysqlclient-1.3.4.tar.gz (77 kB)\n",
      "     |████████████████████████████████| 77 kB 8.7 MB/s             \n",
      "\u001b[?25h  Preparing metadata (setup.py) ... \u001b[?25lerror\n",
      "\u001b[31m  ERROR: Command errored out with exit status 1:\n",
      "   command: /usr/bin/python3 -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '\"'\"'/tmp/pip-install-082ypm9p/mysqlclient_ffa5af58df5944a68c83bb966366e62c/setup.py'\"'\"'; __file__='\"'\"'/tmp/pip-install-082ypm9p/mysqlclient_ffa5af58df5944a68c83bb966366e62c/setup.py'\"'\"';f = getattr(tokenize, '\"'\"'open'\"'\"', open)(__file__) if os.path.exists(__file__) else io.StringIO('\"'\"'from setuptools import setup; setup()'\"'\"');code = f.read().replace('\"'\"'\\r\\n'\"'\"', '\"'\"'\\n'\"'\"');f.close();exec(compile(code, __file__, '\"'\"'exec'\"'\"'))' egg_info --egg-base /tmp/pip-pip-egg-info-yjcya61o\n",
      "       cwd: /tmp/pip-install-082ypm9p/mysqlclient_ffa5af58df5944a68c83bb966366e62c/\n",
      "  Complete output (10 lines):\n",
      "  /bin/sh: 1: mysql_config: not found\n",
      "  Traceback (most recent call last):\n",
      "    File \"<string>\", line 1, in <module>\n",
      "    File \"/tmp/pip-install-082ypm9p/mysqlclient_ffa5af58df5944a68c83bb966366e62c/setup.py\", line 17, in <module>\n",
      "      metadata, options = get_config()\n",
      "    File \"/tmp/pip-install-082ypm9p/mysqlclient_ffa5af58df5944a68c83bb966366e62c/setup_posix.py\", line 47, in get_config\n",
      "      libs = mysql_config(\"libs_r\")\n",
      "    File \"/tmp/pip-install-082ypm9p/mysqlclient_ffa5af58df5944a68c83bb966366e62c/setup_posix.py\", line 29, in mysql_config\n",
      "      raise EnvironmentError(\"%s not found\" % (mysql_config.path,))\n",
      "  OSError: mysql_config not found\n",
      "  ----------------------------------------\u001b[0m\n",
      "\u001b[33mWARNING: Discarding https://files.pythonhosted.org/packages/17/d9/03b08e6a033401335b720806b87596aee8d7f7aa87539713238905c4a8d1/mysqlclient-1.3.4.tar.gz#sha256=a0263383f8ac33ad12d68dc61fa8ccbee41244ce916287c6cee71bf8aea82c5c (from https://pypi.org/simple/mysqlclient/). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.\u001b[0m\n",
      "\u001b[?25h  Downloading mysqlclient-1.3.3.tar.gz (77 kB)\n",
      "     |████████████████████████████████| 77 kB 9.4 MB/s             \n",
      "\u001b[?25h  Preparing metadata (setup.py) ... \u001b[?25lerror\n",
      "\u001b[31m  ERROR: Command errored out with exit status 1:\n",
      "   command: /usr/bin/python3 -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '\"'\"'/tmp/pip-install-082ypm9p/mysqlclient_6f436c919fed4ce2a3279e6a77ceaa1c/setup.py'\"'\"'; __file__='\"'\"'/tmp/pip-install-082ypm9p/mysqlclient_6f436c919fed4ce2a3279e6a77ceaa1c/setup.py'\"'\"';f = getattr(tokenize, '\"'\"'open'\"'\"', open)(__file__) if os.path.exists(__file__) else io.StringIO('\"'\"'from setuptools import setup; setup()'\"'\"');code = f.read().replace('\"'\"'\\r\\n'\"'\"', '\"'\"'\\n'\"'\"');f.close();exec(compile(code, __file__, '\"'\"'exec'\"'\"'))' egg_info --egg-base /tmp/pip-pip-egg-info-hpc0m7sn\n",
      "       cwd: /tmp/pip-install-082ypm9p/mysqlclient_6f436c919fed4ce2a3279e6a77ceaa1c/\n",
      "  Complete output (10 lines):\n",
      "  /bin/sh: 1: mysql_config: not found\n",
      "  Traceback (most recent call last):\n",
      "    File \"<string>\", line 1, in <module>\n",
      "    File \"/tmp/pip-install-082ypm9p/mysqlclient_6f436c919fed4ce2a3279e6a77ceaa1c/setup.py\", line 17, in <module>\n",
      "      metadata, options = get_config()\n",
      "    File \"/tmp/pip-install-082ypm9p/mysqlclient_6f436c919fed4ce2a3279e6a77ceaa1c/setup_posix.py\", line 47, in get_config\n",
      "      libs = mysql_config(\"libs_r\")\n",
      "    File \"/tmp/pip-install-082ypm9p/mysqlclient_6f436c919fed4ce2a3279e6a77ceaa1c/setup_posix.py\", line 29, in mysql_config\n",
      "      raise EnvironmentError(\"%s not found\" % (mysql_config.path,))\n",
      "  OSError: mysql_config not found\n",
      "  ----------------------------------------\u001b[0m\n",
      "\u001b[33mWARNING: Discarding https://files.pythonhosted.org/packages/26/53/c575db342bfca0213c9243ed2642bfadcd420b18d2f477dc812543e3d83a/mysqlclient-1.3.3.tar.gz#sha256=0f0887882d71bf2a079d9724d9323053cfbee8ab2dcfc989daf97d7ce5be9af9 (from https://pypi.org/simple/mysqlclient/). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.\u001b[0m\n",
      "\u001b[?25h  Downloading mysqlclient-1.3.2.tar.gz (77 kB)\n",
      "     |████████████████████████████████| 77 kB 9.5 MB/s             \n",
      "\u001b[?25h  Preparing metadata (setup.py) ... \u001b[?25lerror\n",
      "\u001b[31m  ERROR: Command errored out with exit status 1:\n",
      "   command: /usr/bin/python3 -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '\"'\"'/tmp/pip-install-082ypm9p/mysqlclient_9c61af602ba243ba8d8fe7fcf4ad6ee3/setup.py'\"'\"'; __file__='\"'\"'/tmp/pip-install-082ypm9p/mysqlclient_9c61af602ba243ba8d8fe7fcf4ad6ee3/setup.py'\"'\"';f = getattr(tokenize, '\"'\"'open'\"'\"', open)(__file__) if os.path.exists(__file__) else io.StringIO('\"'\"'from setuptools import setup; setup()'\"'\"');code = f.read().replace('\"'\"'\\r\\n'\"'\"', '\"'\"'\\n'\"'\"');f.close();exec(compile(code, __file__, '\"'\"'exec'\"'\"'))' egg_info --egg-base /tmp/pip-pip-egg-info-7g5riqpe\n",
      "       cwd: /tmp/pip-install-082ypm9p/mysqlclient_9c61af602ba243ba8d8fe7fcf4ad6ee3/\n",
      "  Complete output (10 lines):\n",
      "  /bin/sh: 1: mysql_config: not found\n",
      "  Traceback (most recent call last):\n",
      "    File \"<string>\", line 1, in <module>\n",
      "    File \"/tmp/pip-install-082ypm9p/mysqlclient_9c61af602ba243ba8d8fe7fcf4ad6ee3/setup.py\", line 17, in <module>\n",
      "      metadata, options = get_config()\n",
      "    File \"/tmp/pip-install-082ypm9p/mysqlclient_9c61af602ba243ba8d8fe7fcf4ad6ee3/setup_posix.py\", line 47, in get_config\n",
      "      libs = mysql_config(\"libs_r\")\n",
      "    File \"/tmp/pip-install-082ypm9p/mysqlclient_9c61af602ba243ba8d8fe7fcf4ad6ee3/setup_posix.py\", line 29, in mysql_config\n",
      "      raise EnvironmentError(\"%s not found\" % (mysql_config.path,))\n",
      "  OSError: mysql_config not found\n",
      "  ----------------------------------------\u001b[0m\n",
      "\u001b[33mWARNING: Discarding https://files.pythonhosted.org/packages/2d/df/5440ee86bbb248325cdd6e1fc9cbbba365018a0d2d57f673610e020e6b1d/mysqlclient-1.3.2.tar.gz#sha256=e44025830501b9f70f8c2fe8eeff66f0df2df198802f7295801dac199b8236ef (from https://pypi.org/simple/mysqlclient/). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.\u001b[0m\n",
      "\u001b[?25h  Downloading mysqlclient-1.3.1.tar.gz (76 kB)\n",
      "     |████████████████████████████████| 76 kB 8.8 MB/s             \n",
      "\u001b[?25h  Preparing metadata (setup.py) ... \u001b[?25lerror\n",
      "\u001b[31m  ERROR: Command errored out with exit status 1:\n",
      "   command: /usr/bin/python3 -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '\"'\"'/tmp/pip-install-082ypm9p/mysqlclient_7c98be0bae8d44ad972d8a7aa3678db8/setup.py'\"'\"'; __file__='\"'\"'/tmp/pip-install-082ypm9p/mysqlclient_7c98be0bae8d44ad972d8a7aa3678db8/setup.py'\"'\"';f = getattr(tokenize, '\"'\"'open'\"'\"', open)(__file__) if os.path.exists(__file__) else io.StringIO('\"'\"'from setuptools import setup; setup()'\"'\"');code = f.read().replace('\"'\"'\\r\\n'\"'\"', '\"'\"'\\n'\"'\"');f.close();exec(compile(code, __file__, '\"'\"'exec'\"'\"'))' egg_info --egg-base /tmp/pip-pip-egg-info-bdxr9ubk\n",
      "       cwd: /tmp/pip-install-082ypm9p/mysqlclient_7c98be0bae8d44ad972d8a7aa3678db8/\n",
      "  Complete output (10 lines):\n",
      "  /bin/sh: 1: mysql_config: not found\n",
      "  Traceback (most recent call last):\n",
      "    File \"<string>\", line 1, in <module>\n",
      "    File \"/tmp/pip-install-082ypm9p/mysqlclient_7c98be0bae8d44ad972d8a7aa3678db8/setup.py\", line 17, in <module>\n",
      "      metadata, options = get_config()\n",
      "    File \"/tmp/pip-install-082ypm9p/mysqlclient_7c98be0bae8d44ad972d8a7aa3678db8/setup_posix.py\", line 47, in get_config\n",
      "      libs = mysql_config(\"libs_r\")\n",
      "    File \"/tmp/pip-install-082ypm9p/mysqlclient_7c98be0bae8d44ad972d8a7aa3678db8/setup_posix.py\", line 29, in mysql_config\n",
      "      raise EnvironmentError(\"%s not found\" % (mysql_config.path,))\n",
      "  OSError: mysql_config not found\n",
      "  ----------------------------------------\u001b[0m\n",
      "\u001b[33mWARNING: Discarding https://files.pythonhosted.org/packages/6b/ba/4729d99e85a0a35bb46d55500570de05b4af10431cef174b6da9f58a0e50/mysqlclient-1.3.1.tar.gz#sha256=3549e8a61f10c8cd8eac6581d3f44d0594f535fb7b29e6090db3a0bc547b25ad (from https://pypi.org/simple/mysqlclient/). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.\u001b[0m\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[?25h  Downloading mysqlclient-1.3.0.tar.gz (76 kB)\n",
      "     |████████████████████████████████| 76 kB 9.1 MB/s             \n",
      "\u001b[?25h  Preparing metadata (setup.py) ... \u001b[?25lerror\n",
      "\u001b[31m  ERROR: Command errored out with exit status 1:\n",
      "   command: /usr/bin/python3 -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '\"'\"'/tmp/pip-install-082ypm9p/mysqlclient_8c63c04148ca430cacb6612a1c2a0a99/setup.py'\"'\"'; __file__='\"'\"'/tmp/pip-install-082ypm9p/mysqlclient_8c63c04148ca430cacb6612a1c2a0a99/setup.py'\"'\"';f = getattr(tokenize, '\"'\"'open'\"'\"', open)(__file__) if os.path.exists(__file__) else io.StringIO('\"'\"'from setuptools import setup; setup()'\"'\"');code = f.read().replace('\"'\"'\\r\\n'\"'\"', '\"'\"'\\n'\"'\"');f.close();exec(compile(code, __file__, '\"'\"'exec'\"'\"'))' egg_info --egg-base /tmp/pip-pip-egg-info-jzqdbxa6\n",
      "       cwd: /tmp/pip-install-082ypm9p/mysqlclient_8c63c04148ca430cacb6612a1c2a0a99/\n",
      "  Complete output (10 lines):\n",
      "  /bin/sh: 1: mysql_config: not found\n",
      "  Traceback (most recent call last):\n",
      "    File \"<string>\", line 1, in <module>\n",
      "    File \"/tmp/pip-install-082ypm9p/mysqlclient_8c63c04148ca430cacb6612a1c2a0a99/setup.py\", line 17, in <module>\n",
      "      metadata, options = get_config()\n",
      "    File \"/tmp/pip-install-082ypm9p/mysqlclient_8c63c04148ca430cacb6612a1c2a0a99/setup_posix.py\", line 47, in get_config\n",
      "      libs = mysql_config(\"libs_r\")\n",
      "    File \"/tmp/pip-install-082ypm9p/mysqlclient_8c63c04148ca430cacb6612a1c2a0a99/setup_posix.py\", line 29, in mysql_config\n",
      "      raise EnvironmentError(\"%s not found\" % (mysql_config.path,))\n",
      "  OSError: mysql_config not found\n",
      "  ----------------------------------------\u001b[0m\n",
      "\u001b[33mWARNING: Discarding https://files.pythonhosted.org/packages/6a/91/bdfe808fb5dc99a5f65833b370818161b77ef6d1e19b488e4c146ab615aa/mysqlclient-1.3.0.tar.gz#sha256=06eb5664e3738b283ea2262ee60ed83192e898f019cc7ff251f4d05a564ab3b7 (from https://pypi.org/simple/mysqlclient/). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.\u001b[0m\n",
      "\u001b[?25hCollecting mysql\n",
      "  Downloading mysql-0.0.2.tar.gz (1.9 kB)\n",
      "  Preparing metadata (setup.py) ... \u001b[?25ldone\n",
      "\u001b[?25h  Downloading mysql-0.0.1.tar.gz (1.7 kB)\n",
      "  Preparing metadata (setup.py) ... \u001b[?25ldone\n",
      "\u001b[?25hCollecting MySQL-python\n",
      "  Downloading MySQL-python-1.2.5.zip (108 kB)\n",
      "     |████████████████████████████████| 108 kB 65.1 MB/s            \n",
      "\u001b[?25h  Preparing metadata (setup.py) ... \u001b[?25lerror\n",
      "\u001b[31m  ERROR: Command errored out with exit status 1:\n",
      "   command: /usr/bin/python3 -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '\"'\"'/tmp/pip-install-082ypm9p/mysql-python_79496d8a453d4b5aa807251711cda857/setup.py'\"'\"'; __file__='\"'\"'/tmp/pip-install-082ypm9p/mysql-python_79496d8a453d4b5aa807251711cda857/setup.py'\"'\"';f = getattr(tokenize, '\"'\"'open'\"'\"', open)(__file__) if os.path.exists(__file__) else io.StringIO('\"'\"'from setuptools import setup; setup()'\"'\"');code = f.read().replace('\"'\"'\\r\\n'\"'\"', '\"'\"'\\n'\"'\"');f.close();exec(compile(code, __file__, '\"'\"'exec'\"'\"'))' egg_info --egg-base /tmp/pip-pip-egg-info-291zq32_\n",
      "       cwd: /tmp/pip-install-082ypm9p/mysql-python_79496d8a453d4b5aa807251711cda857/\n",
      "  Complete output (7 lines):\n",
      "  Traceback (most recent call last):\n",
      "    File \"<string>\", line 1, in <module>\n",
      "    File \"/tmp/pip-install-082ypm9p/mysql-python_79496d8a453d4b5aa807251711cda857/setup.py\", line 13, in <module>\n",
      "      from setup_posix import get_config\n",
      "    File \"/tmp/pip-install-082ypm9p/mysql-python_79496d8a453d4b5aa807251711cda857/setup_posix.py\", line 2, in <module>\n",
      "      from ConfigParser import SafeConfigParser\n",
      "  ModuleNotFoundError: No module named 'ConfigParser'\n",
      "  ----------------------------------------\u001b[0m\n",
      "\u001b[33mWARNING: Discarding https://files.pythonhosted.org/packages/a5/e9/51b544da85a36a68debe7a7091f068d802fc515a3a202652828c73453cad/MySQL-python-1.2.5.zip#sha256=811040b647e5d5686f84db415efd697e6250008b112b6909ba77ac059e140c74 (from https://pypi.org/simple/mysql-python/). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.\u001b[0m\n",
      "\u001b[?25h  Downloading MySQL-python-1.2.4.zip (113 kB)\n",
      "     |████████████████████████████████| 113 kB 72.7 MB/s            \n",
      "\u001b[?25h  Preparing metadata (setup.py) ... \u001b[?25lerror\n",
      "\u001b[31m  ERROR: Command errored out with exit status 1:\n",
      "   command: /usr/bin/python3 -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '\"'\"'/tmp/pip-install-082ypm9p/mysql-python_d0ad5f7052a14194a9669f25d7a5258f/setup.py'\"'\"'; __file__='\"'\"'/tmp/pip-install-082ypm9p/mysql-python_d0ad5f7052a14194a9669f25d7a5258f/setup.py'\"'\"';f = getattr(tokenize, '\"'\"'open'\"'\"', open)(__file__) if os.path.exists(__file__) else io.StringIO('\"'\"'from setuptools import setup; setup()'\"'\"');code = f.read().replace('\"'\"'\\r\\n'\"'\"', '\"'\"'\\n'\"'\"');f.close();exec(compile(code, __file__, '\"'\"'exec'\"'\"'))' egg_info --egg-base /tmp/pip-pip-egg-info-p39jty6g\n",
      "       cwd: /tmp/pip-install-082ypm9p/mysql-python_d0ad5f7052a14194a9669f25d7a5258f/\n",
      "  Complete output (31 lines):\n",
      "  Downloading http://pypi.python.org/packages/source/d/distribute/distribute-0.6.28.tar.gz\n",
      "  Traceback (most recent call last):\n",
      "    File \"/tmp/pip-install-082ypm9p/mysql-python_d0ad5f7052a14194a9669f25d7a5258f/distribute_setup.py\", line 143, in use_setuptools\n",
      "      raise ImportError\n",
      "  ImportError\n",
      "  \n",
      "  During handling of the above exception, another exception occurred:\n",
      "  \n",
      "  Traceback (most recent call last):\n",
      "    File \"<string>\", line 1, in <module>\n",
      "    File \"/tmp/pip-install-082ypm9p/mysql-python_d0ad5f7052a14194a9669f25d7a5258f/setup.py\", line 7, in <module>\n",
      "      use_setuptools()\n",
      "    File \"/tmp/pip-install-082ypm9p/mysql-python_d0ad5f7052a14194a9669f25d7a5258f/distribute_setup.py\", line 145, in use_setuptools\n",
      "      return _do_download(version, download_base, to_dir, download_delay)\n",
      "    File \"/tmp/pip-install-082ypm9p/mysql-python_d0ad5f7052a14194a9669f25d7a5258f/distribute_setup.py\", line 124, in _do_download\n",
      "      to_dir, download_delay)\n",
      "    File \"/tmp/pip-install-082ypm9p/mysql-python_d0ad5f7052a14194a9669f25d7a5258f/distribute_setup.py\", line 194, in download_setuptools\n",
      "      src = urlopen(url)\n",
      "    File \"/usr/lib/python3.6/urllib/request.py\", line 223, in urlopen\n",
      "      return opener.open(url, data, timeout)\n",
      "    File \"/usr/lib/python3.6/urllib/request.py\", line 532, in open\n",
      "      response = meth(req, response)\n",
      "    File \"/usr/lib/python3.6/urllib/request.py\", line 642, in http_response\n",
      "      'http', request, response, code, msg, hdrs)\n",
      "    File \"/usr/lib/python3.6/urllib/request.py\", line 570, in error\n",
      "      return self._call_chain(*args)\n",
      "    File \"/usr/lib/python3.6/urllib/request.py\", line 504, in _call_chain\n",
      "      result = func(*args)\n",
      "    File \"/usr/lib/python3.6/urllib/request.py\", line 650, in http_error_default\n",
      "      raise HTTPError(req.full_url, code, msg, hdrs, fp)\n",
      "  urllib.error.HTTPError: HTTP Error 403: SSL is required\n",
      "  ----------------------------------------\u001b[0m\n",
      "\u001b[33mWARNING: Discarding https://files.pythonhosted.org/packages/90/5a/ce7bef80825c2188cf507baf57b37516e18dffdb198a6766a597f703059a/MySQL-python-1.2.4.zip#sha256=e405f9d6be33923d428acaa4db4f4470427f1d15ea0d2d82a933449ace26bbd9 (from https://pypi.org/simple/mysql-python/). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.\u001b[0m\n",
      "\u001b[?25h  Downloading MySQL-python-1.2.3.tar.gz (70 kB)\n",
      "     |████████████████████████████████| 70 kB 8.7 MB/s             \n",
      "\u001b[?25h  Preparing metadata (setup.py) ... \u001b[?25lerror\n",
      "\u001b[31m  ERROR: Command errored out with exit status 1:\n",
      "   command: /usr/bin/python3 -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '\"'\"'/tmp/pip-install-082ypm9p/mysql-python_1b1500a1e6c34b89854701d4f4bf550b/setup.py'\"'\"'; __file__='\"'\"'/tmp/pip-install-082ypm9p/mysql-python_1b1500a1e6c34b89854701d4f4bf550b/setup.py'\"'\"';f = getattr(tokenize, '\"'\"'open'\"'\"', open)(__file__) if os.path.exists(__file__) else io.StringIO('\"'\"'from setuptools import setup; setup()'\"'\"');code = f.read().replace('\"'\"'\\r\\n'\"'\"', '\"'\"'\\n'\"'\"');f.close();exec(compile(code, __file__, '\"'\"'exec'\"'\"'))' egg_info --egg-base /tmp/pip-pip-egg-info-n6ijz9wr\n",
      "       cwd: /tmp/pip-install-082ypm9p/mysql-python_1b1500a1e6c34b89854701d4f4bf550b/\n",
      "  Complete output (8 lines):\n",
      "  Traceback (most recent call last):\n",
      "    File \"<string>\", line 1, in <module>\n",
      "    File \"/tmp/pip-install-082ypm9p/mysql-python_1b1500a1e6c34b89854701d4f4bf550b/setup.py\", line 11, in <module>\n",
      "      from setup_posix import get_config\n",
      "    File \"/tmp/pip-install-082ypm9p/mysql-python_1b1500a1e6c34b89854701d4f4bf550b/setup_posix.py\", line 101\n",
      "      print \"\"\"You shouldn't be running this directly; it is used by setup.py.\"\"\"\n",
      "                                                                                ^\n",
      "  SyntaxError: Missing parentheses in call to 'print'. Did you mean print(\"\"\"You shouldn't be running this directly)?\n",
      "  ----------------------------------------\u001b[0m\n",
      "\u001b[33mWARNING: Discarding https://files.pythonhosted.org/packages/9a/81/924d6799494cf7fb24370335c2f782088d6ac4f79e4137d4df94cea3582c/MySQL-python-1.2.3.tar.gz#sha256=7de66fbbf923634e7c965aeaefa74642ba75ae20ee1cefcefc3009595b7a7e6e (from https://pypi.org/simple/mysql-python/). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.\u001b[0m\n",
      "\u001b[31mERROR: Cannot install mysql==0.0.1, mysql==0.0.2 and mysql==0.0.3 because these package versions have conflicting dependencies.\u001b[0m\n",
      "\u001b[?25h\n",
      "The conflict is caused by:\n",
      "    mysql 0.0.3 depends on mysqlclient\n",
      "    mysql 0.0.2 depends on mysqlclient\n",
      "    mysql 0.0.1 depends on MySQL-python\n",
      "\n",
      "To fix this you could try to:\n",
      "1. loosen the range of package versions you've specified\n",
      "2. remove package versions to allow pip attempt to solve the dependency conflict\n",
      "\n",
      "\u001b[31mERROR: ResolutionImpossible: for help visit https://pip.pypa.io/en/latest/user_guide/#fixing-conflicting-dependencies\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "!pip install mysql"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "id": "ac33fb6b",
   "metadata": {},
   "outputs": [],
   "source": [
    "import mysql.connector"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 25,
   "id": "7acacc6e",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "<mysql.connector.connection.MySQLConnection object at 0x7f7471c21320>\n"
     ]
    }
   ],
   "source": [
    "import mysql.connector\n",
    "\n",
    "mydb = mysql.connector.connect(\n",
    "  host=\"localhost\",\n",
    "  user=\"ian\",\n",
    "    database='test'\n",
    ")\n",
    "\n",
    "print(mydb)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 28,
   "id": "72c414b0",
   "metadata": {},
   "outputs": [],
   "source": [
    "mycursor = mydb.cursor()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "id": "076de4b4",
   "metadata": {
    "scrolled": true
   },
   "outputs": [],
   "source": [
    "mycursor.execute(\"CREATE DATABASE test\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 29,
   "id": "02cdff2c",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('information_schema',)\n",
      "('mysql',)\n",
      "('performance_schema',)\n",
      "('postthread',)\n",
      "('postthread2',)\n",
      "('sys',)\n",
      "('test',)\n"
     ]
    }
   ],
   "source": [
    "mycursor.execute(\"SHOW DATABASES\")\n",
    "\n",
    "for x in mycursor:\n",
    "  print(x)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 30,
   "id": "c64c914a",
   "metadata": {},
   "outputs": [],
   "source": [
    "mycursor.execute(\"CREATE TABLE customers (name VARCHAR(255), address VARCHAR(255))\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 36,
   "id": "cc16315a",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "ename": "ProgrammingError",
     "evalue": "1064 (42000): You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'INT)' at line 1",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mProgrammingError\u001b[0m                          Traceback (most recent call last)",
      "\u001b[0;32m<ipython-input-36-78fe8eca6391>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m\u001b[0m\n\u001b[0;32m----> 1\u001b[0;31m \u001b[0mmycursor\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mexecute\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m\"CREATE TABLE messages (data TEXT, signer VARCHAR(255), msa_id INT, index INT)\"\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m",
      "\u001b[0;32m~/.local/lib/python3.6/site-packages/mysql/connector/cursor.py\u001b[0m in \u001b[0;36mexecute\u001b[0;34m(self, operation, params, multi)\u001b[0m\n\u001b[1;32m    570\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    571\u001b[0m         \u001b[0;32mtry\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 572\u001b[0;31m             \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0m_handle_result\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0m_connection\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mcmd_query\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mstmt\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    573\u001b[0m         \u001b[0;32mexcept\u001b[0m \u001b[0merrors\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mInterfaceError\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    574\u001b[0m             \u001b[0;32mif\u001b[0m \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0m_connection\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0m_have_next_result\u001b[0m\u001b[0;34m:\u001b[0m  \u001b[0;31m# pylint: disable=W0212\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;32m~/.local/lib/python3.6/site-packages/mysql/connector/connection.py\u001b[0m in \u001b[0;36mcmd_query\u001b[0;34m(self, query, raw, buffered, raw_as_string)\u001b[0m\n\u001b[1;32m    920\u001b[0m         \u001b[0mquery\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mbytes\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mpacket\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    921\u001b[0m         \u001b[0;32mtry\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 922\u001b[0;31m             \u001b[0mresult\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0m_handle_result\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0m_send_cmd\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mServerCmd\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mQUERY\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mquery\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    923\u001b[0m         \u001b[0;32mexcept\u001b[0m \u001b[0merrors\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mProgrammingError\u001b[0m \u001b[0;32mas\u001b[0m \u001b[0merr\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    924\u001b[0m             \u001b[0;32mif\u001b[0m \u001b[0merr\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0merrno\u001b[0m \u001b[0;34m==\u001b[0m \u001b[0;36m3948\u001b[0m \u001b[0;32mand\u001b[0m\u001b[0;31m \u001b[0m\u001b[0;31m\\\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;32m~/.local/lib/python3.6/site-packages/mysql/connector/connection.py\u001b[0m in \u001b[0;36m_handle_result\u001b[0;34m(self, packet)\u001b[0m\n\u001b[1;32m    730\u001b[0m             \u001b[0;32mreturn\u001b[0m \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0m_handle_eof\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mpacket\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    731\u001b[0m         \u001b[0;32melif\u001b[0m \u001b[0mpacket\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0;36m4\u001b[0m\u001b[0;34m]\u001b[0m \u001b[0;34m==\u001b[0m \u001b[0;36m255\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 732\u001b[0;31m             \u001b[0;32mraise\u001b[0m \u001b[0merrors\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mget_exception\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mpacket\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    733\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    734\u001b[0m         \u001b[0;31m# We have a text result set\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;31mProgrammingError\u001b[0m: 1064 (42000): You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'INT)' at line 1"
     ]
    }
   ],
   "source": [
    "mycursor.execute(\"CREATE TABLE messages (data TEXT, signer VARCHAR(255), msa_id INT, index INT)\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "ee3d942f",
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.6.9"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
